app/src/main/java/com/duycuong/weather/data/source/remote/WeatherRemoteDataSource.java

```diff
@@ -0,0 +1,127 @@
+package com.duycuong.weather.data.source.remote;
+
+import android.content.res.Resources;
+import android.location.Location;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports